src/components/QuestionnaireForm/QuestionnaireForm.css

```diff
@@ -1,12 +1,24 @@
+.questionaire-form {
+  margin-left: 55px;
+  margin-top: 1em;
+}
+
+.questionaire-form .section {
+  margin-left: 15px;
+}
+
 .wrapper{
-  display:block;
-  margin-left:15px;
+  /* display:block; */
```

Comment:
Should all of these commented out lines be removed?

Reply:
@hkong did you see Pat's question?

Reply:
@plarocque4 sorry for the delayed response, not sure why I didn't see your question until now.  Yes, I'll remove them in the latest commit.